selftests/unit/test_datadir.py

```diff
@@ -51,16 +51,11 @@ def test_datadir_from_config(self):
         When avocado.conf is present, honor the values coming from it.
         """
         stg = settings.Settings(self.config_file_path)
-        # Trick the module to think we're on a system wide install
-        stg.intree = False
```

Comment:
I think removing it is a good idea indeed because:
* Few people (should) run Avocado from source trees AND need to have custom config
* The defaults are being moved out of the configuration files